content/docs/user-guide/components/reference/atom/bloom.md

```diff
@@ -1,20 +1,39 @@
 ---
+linkTitle: Bloom
 title: Bloom Component
-linktitle: Bloom
-description: "Open 3D Engine (O3DE) Bloom component reference."
+description: The Bloom component, which simulates real-world light bleeding (glow), is provided by the Atom Gem in Open 3D Engine (O3DE). 
 toc: true
 ---
 
-The **Bloom component** simulates real-world light bleeding, or glow.
+The **Bloom** component creates *bloom*, a post-processing effect that simulates real-world light bleeding, or glow.
 
 
-## Provider ##
+## Provider
 
-[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom)
+
+
+## Dependencies
+
+[PostFX Layer component](./postfx-layer)
 
 
 ## Properties
 
-{{< todo issue=https://github.com/o3de/o3de.org/issues/717 >}}
-Add properties.
-{{< /todo >}}
\ No newline at end of file
+![Bloom component interface](/images/user-guide/components/reference/atom/bloom-component-ui.png)
+
+
+### Bloom properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Enable Bloom** | If enabled, activates the bloom effect. | Boolean | `Enabled` |
```

Comment:
Oddly enough, when I first add the Bloom component, it's disabled.
```suggestion
| **Enable Bloom** | If enabled, activates the bloom effect. | Boolean | `Disabled` |
```

Reply:
I thought this was odd too. Thanks for catching this!